### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --format brief`

### CODE REVIEW

File: crates/filterer/globset/tests/filtering.rs [hunk 1]
```diff
@@ -591,7 +591,7 @@ async fn ignore_folder_incorrectly_with_only_double_double_glob() {
 	filterer.dir_doesnt_pass("prunes/carrots/cauliflowers/oranges");
 	filterer.dir_doesnt_pass("prunes/carrots/cauliflowers/artichokes/oranges");
 
-	// buggy behaviour (should be doesnt):
+	// buggy behaviour (should be doesn't):
```
Comment (passcod) [inline crates/filterer/globset/tests/filtering.rs:594]: those aren't, they're literally referring to the method name

File: doc/watchexec.1.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ directories.
 
 :   Clear screen before running command
 
-If this doesnt completely clear the screen, try \--clear=reset.
+If this doesn't completely clear the screen, try \--clear=reset.
```
Comment (passcod) [inline doc/watchexec.1.md:81]: all the ones in this file should be fixed in args.rs, this file is generated

**Comment by passcod** [comment]
Thanks, but also a good example of needing to manually review the output of automated tools

File: crates/filterer/globset/tests/filtering.rs [hunk 3]
```diff
@@ -556,7 +556,7 @@ async fn ignore_folder_incorrectly_with_bare_and_trailing_slash() {
 
 	filterer.dir_doesnt_pass("prunes");
 
-	// buggy behaviour (should be doesnt):
+	// buggy behaviour (should be doesn't):
```
Comment (passcod) [inline crates/filterer/globset/tests/filtering.rs:559]: I really did mean that all of the ones in this file and other test files are not typos.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**